### PR TITLE
Extract bridgeMap into own package to improve testability

### PR DESCRIPTION
**Pull request by patcon**
Right now, starting the router loads all bridges in gateway package, which means that we'll get import cycles if we're including this package in `*_test.go` for bridger tests. This addresses that in what seems like a clean way. When creating a new TestRouter, we can just pass in an isolated bridgeMap into `NewRouter()` with the only the bridge we're testing :)

Ready for review 🎉

### CODE REVIEW

File: gateway/bmap/bmap.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+	birc "github.com/42wim/matterbridge/bridge/irc"
+	bmatrix "github.com/42wim/matterbridge/bridge/matrix"
+	bmattermost "github.com/42wim/matterbridge/bridge/mattermost"
+	brocketchat "github.com/42wim/matterbridge/bridge/rocketchat"
+	bslack "github.com/42wim/matterbridge/bridge/slack"
+	bsshchat "github.com/42wim/matterbridge/bridge/sshchat"
+	bsteam "github.com/42wim/matterbridge/bridge/steam"
+	btelegram "github.com/42wim/matterbridge/bridge/telegram"
+	bxmpp "github.com/42wim/matterbridge/bridge/xmpp"
+	bzulip "github.com/42wim/matterbridge/bridge/zulip"
```
[the remark this answers is not in the document]
Comment (patcon) [reply gateway/bmap/bmap.go]: Done! thanks

**Comment by patcon** [comment]
Also changed package/file names a bit so that browsing file names was more self-explanatory (bmap seemed opaque)

Ready for review

File: matterbridge.go [hunk 2]
```diff
@@ -8,6 +8,8 @@ import (
 
 	"github.com/42wim/matterbridge/bridge/config"
 	"github.com/42wim/matterbridge/gateway"
+	"github.com/42wim/matterbridge/gateway/bridgemap"
+
```
Comment (Helcaraxan) [inline matterbridge.go]: Nit: unnecessary empty line.

**Comment by patcon** [comment]
This is ready for final review.

**Pull request merged**